### PR TITLE
Optimize the jobflow architecture design diagram

### PR DESCRIPTION
**Pull request by wangyang0616**
Adopt a more concise color scheme and optimize the color and style of the architecture diagram.

### CODE REVIEW

**Review by Thor-wl — APPROVED** [review]
@wangyang0616 Thanks for your hard work! The new architecture turns to be more clear than the older ones. They not only highlight the added compontents but also mark the resource relationship.

**Review by hwdef — COMMENTED** [review]
/hold

File: docs/design/jobflow/README.md [hunk 1]
```diff
@@ -24,15 +24,13 @@ JobFlow helps migrating AI, BigData, HPC workloads to the cloud-native world. Th
...
 
-![jobflow-2.jpg](../images/jobflow-2.jpg)
+![jobflow-1.png](../images/jobflow-1.png)
 
 ## Design
 
-![jobflow-1.png](../images/jobflow-1.png)
-
-![jobflow-3.png](../images/jobflow-3.png)
+![jobflow-1.png](../images/jobflow-2.png)
```
Comment (hwdef) [inline docs/design/jobflow/README.md:31]:
```suggestion
![jobflow-2.png](../images/jobflow-2.png)
```

File: docs/design/jobflow/README.md [hunk 2]
```diff
@@ -58,7 +56,7 @@ Deleting a jobflow when the jobflow is in a non-complete state will be intercept
 
 ### Controller
 
-![jobflow-4.png](../images/jobflow-4.png)
+![jobflow-4.png](../images/jobflow-3.png)
```
Comment (hwdef) [inline docs/design/jobflow/README.md:59]:
```suggestion
![jobflow-3.png](../images/jobflow-3.png)
```

File: docs/design/jobflow/README.md (hunk 1 above)
Comment (wangyang0616) [reply docs/design/jobflow/README.md]: done

File: docs/design/jobflow/README.md (hunk 2 above)
Comment (wangyang0616) [reply docs/design/jobflow/README.md]: done

**Review by Thor-wl — COMMENTED** [review]
/lgtm

**Review by hwdef — COMMENTED** [review]
/lgtm
/hold cancel

File: docs/design/jobflow/README.md [hunk 3]
```diff
@@ -24,15 +24,13 @@ JobFlow helps migrating AI, BigData, HPC workloads to the cloud-native world. Th
...
 
 ## Design
 
-![jobflow-1.png](../images/jobflow-1.png)
+![jobflow-2.png](../images/jobflow-2.png)
 
-![jobflow-3.png](../images/jobflow-3.png)
-
-The blue part is the component of k8s itself, the green and brown are the components of volcano, and the yellow is the crd resource of volcano.
+The blue part is the components of k8s itself, the orange is the existing definition of Volcano, and the red is the new definition of JobFlow.
```
Comment (william-wang) [inline docs/design/jobflow/README.md:33]: please update the description. It is out of date after refreshing the jobflow-2.jpg

**Pull request merged**